packages/ra-ui-materialui/src/list/filter/FilterLiveSearch.tsx

```diff
@@ -74,9 +78,10 @@ export const FilterLiveSearch = memo((props: FilterLiveSearchProps) => {
     );
 });
 
-export interface FilterLiveSearchProps {
+export interface FilterLiveSearchProps extends Omit<TextInputProps, 'source'> {
     source?: string;
     sx?: SxProps;
+    label?: string;
```

Comment:
I don't think we should override the label prop as we forward it as is to the `TextInput`. We only provide a default

Reply:
I would tend to agree with @septentrion-730n because his solution still provides a default value and also supports being overridden by the user directly in the FilterLiveSearch component.
Previously the only way you could customize the label was through localization.

However, it could be nice to handle translation on this field either way.

Reply:
The `TextInput` already support all these use cases: https://github.com/marmelab/react-admin/blob/master/packages/ra-ui-materialui/src/input/TextInput.tsx#L66 via `FieldTitle`. 

Btw, we don't event need to translate here, we could just pass the translation key directly as `label` and it will be translated